docs/pages/04.02-callbacks.md

```diff
@@ -114,6 +114,23 @@ buttonIconHandler(data, callbackIcon) {
 }
 ```
 
+### propertyIconHandler()
+Called when a user wants to pass in a specific icon to a dropdown menu. The propertyIconHandler expects a Carbon Icon jsx object as the return value from the callback. This is used to display the Carbon icon in the dropdown menu. The propertyIconHandler provides the following data:
+
+-data: an object that consists of
+  - propertyId: of the dropdown that was selected
+  - enumValue: of the dropdown that was selected
+
+```js
+propertyIconHandler(data, callbackIcon) {
+  const propertyId = { "name": "oneofselect" };
+	const { iconSwitch } = this.state;
+	if (iconSwitch === true && data.propertyId.name === propertyId.name) {
+		callbackIcon(<Password />);
```

Comment:
I would rename `Password` to `Icon` or something that makes it more clear it's an icon users should pass in.